rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -125,17 +128,77 @@ def __init__(self, pipeline, tensor_layout = types.NCHW, reverse_channels = Fals
             self.len = b.getRemainingImages(self.loader._handle)//self.bs
         else:
             self.len = b.getRemainingImages(self.loader._handle)
+        self.num_batches = self.loader._external_source.n //self.bs if self.loader._external_source.n%self.bs == 0 else (self.loader._external_source.n //self.bs + 1)
```

Comment:
Please check if this changes are needed in pytorch.py
since this is a generic iterator used for all pipelines

Reply:
@swetha097 Please check this